### PR TITLE
add test forward pass

### PR DESCRIPTION
**Pull request by pengbo-learn**
Forward pass in test model should fix running mean and running var in batch normalization op.
So another forward pass is needed.

### CODE REVIEW

**Comment by orionr** [comment]
Thank you for your contribution! PyTorch and Caffe2 are now officially merged with the PyTorch 1.0 preview release. Please submit on https://github.com/pytorch/pytorch if this is still an issue over there.
